packages/host/src/Admin/Settings.tsx

```diff
@@ -37,13 +37,13 @@ export const Settings: React.FC = () => {
       setCustomTheme(themeOptions);
       navigate(0);
     } catch (e) {
-      error(`${t('error.something-wrong')} ${e.message}`)();
```

Comment:
some type errors that showed up.. think these are ok though I didn't check too deeply..